### ISSUE TITLE
Unable to view schemas in schema registry

### ISSUE THREAD

**Issue by qmarsala-cricut** [provectus/kafka-ui#2196]
**Describe the bug**
I have added a protobuf schema to the registry that imports googles timestamp type.  When loading the schema registry ui, I now get a 404 because it cannot find the imported proto definition.  While this is somewhat expected, since I found out confluents registry wont be natively supporting google well known types until 6.2.0, it is frustrating that we can no longer see any schema, as long as at least one cannot be read.

It would be nice if the other schemas could still be seen, and maybe even the schema with an issue could still be shown so you can delete or edit the schema.

**Set up**
using v0.4.0 - 521ba0c

**Steps to Reproduce**

1. create a new proto schema with the following line
```import "google/protobuf/timestamp.proto";```
2. go to schema registry in kafka ui

observe 404 error toast notification and no schemas shown

**Expected behavior**

see the working schemas, maybe even the bad one with some sort of indication it has a problem

**Additional context**
Any known workarounds would be great, and this may not be a problem soon once confluent supports these types in the registry natively.

**Comment by Haarolean** [comment]
Hey, which SR version are you on?
I tried with `confluentinc/cp-schema-registry:5.5.0`, it seems to be working fine:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/1494347/174841431-41a5dd14-6018-4420-bd43-8bf6f19d1641.png">

**Comment by qmarsala-cricut** [comment]
with sr: `confluentinc/cp-schema-registry:6.1.0`
and schema: 
```
syntax = "proto3";
import "google/protobuf/timestamp.proto";
package testing;

message Notification {
    google.protobuf.Timestamp PublishTime = 1;
}
```

My application was able to register the schema which then broke my ability to view the schemas.
However, when trying to manually add this through the ui I now get 422: unprocessable entity

**Comment by qmarsala-cricut** [comment]
I am also unable to reproduce this by hand.  I am redeploying our broken build again to reproduce it that way.  I will let you know once that completes how it goes.
The code base is dotnet and uses the Confluent libraries:
Confluent.Kafka
Confluent.SchemaRegistry
Confluent.SchemaRegistry.Serdes.Protobuf

**Comment by qmarsala-cricut** [comment]
I'm not sure that I can, since "google/protobuf/timestamp.proto" is a well known type and not mine.
When I tried to change my proto file in my solution, it could no longer compile the notification type since it could no longer resolve timestamp.proto with a different path. 

a potential work around could be to add our own version of timestamp.proto, and then change the import to utilize that?  However, we do not want to have to maintain this type ourselves, as it also has methods our code relies on provided by google already.

**Comment by Haarolean** [comment]
Nah, don't worry about the workarounds, it's actually a bug on our side (#1995). 
I'll close this one as a duplicate, feel free to subscribe to #1995, we're planning to fix that one soon.
